app/worker.php

```diff
@@ -100,7 +100,7 @@
 $pools = $register->get('pools');
 $connection = $pools->get('queue')->pop()->getResource();
 $workerNumber = swoole_cpu_num() * intval(App::getEnv('_APP_WORKER_PER_CORE', 6));
-
+$workerNumber = 1;
```

Comment:
Please avoid committing this change, it's much better to change it in your local env as this can be very risky if it gets merged.